test/fail_compilation/fail155.d

```diff
@@ -1,7 +1,8 @@
 /*
 TEST_OUTPUT:
 ---
-fail_compilation/fail155.d(19): Error: overlapping initialization for `y`
+fail_compilation/fail155.d(20): Error: overlapping initialization for `y`
+fail_compilation/fail155.d(20):        struct initializers for `struct`s that contain anonymous `union`s must initialize only the first member of a `union`. All subsequent non-overlapping fields are default initialized
```

Comment:
Nitpick: Typically DMD only quotes symbol names in backticks, like `y` above. 

I think you can just use `structs` instead of `'struct's`, because it almost reads like a possessive form (`struct's`, as in belonging to some struct, rather than the plural `structs`).